packages/server/tests/acceptance/htsPrecompile.spec.ts

```diff
@@ -361,4 +340,96 @@ describe('HTS Precompile Acceptance Tests', async function () {
       expect(pauseStatusAfter).to.equal(false);
     });
   });
+
+  // FIXME
+  // Requires a newer version of services than 0.29.0-alpha.1
+  xdescribe('HTS Precompile Wipe Tests', async function() {
+
+    before(async function() {
+      const amount = 5;
+      await baseHTSContract.transferTokenPublic(accounts[1].wallet.address, HTSTokenContractAddress, amount);
+    });
+
+    it('should revert if attempting to wipe more tokens than the owned amount', async function() {
+      const wipeAmount = 100;
+      const balanceBefore = await HTSTokenContract.balanceOf(accounts[1].wallet.address);
+
+      const tx = await baseHTSContract.wipeTokenAccountPublic(HTSTokenContractAddress, accounts[1].wallet.address, wipeAmount, { gasLimit: 1_000_000 });
+
+      await Assertions.expectRevert(tx, 'CALL_EXCEPTION');
+      const balanceAfter = await HTSTokenContract.balanceOf(accounts[1].wallet.address);
+      expect(balanceBefore.toString()).to.eq(balanceAfter.toString());
+    });
+
+    it('should be able to execute wipeTokenAccount', async function() {
+      const wipeAmount = 3;
+      const balanceBefore = await HTSTokenContract.balanceOf(accounts[1].wallet.address);
+
+      const tx = await baseHTSContract.wipeTokenAccountPublic(HTSTokenContractAddress, accounts[1].wallet.address, wipeAmount, { gasLimit: 1_000_000 });
+      const { responseCode } = (await tx.wait()).events.filter(e => e.event === 'ResponseCode')[0].args;
+      expect(responseCode).to.equal(TX_SUCCESS_CODE);
+
+      const balanceAfter = await HTSTokenContract.balanceOf(accounts[1].wallet.address);
+      expect(Number(balanceAfter.toString()) + wipeAmount).to.equal(Number(balanceBefore.toString()));
+    });
+
+    it('should be able to execute wipeTokenAccountNFT', async function() {
+      let NftSerialNumber, serials;
+
+      // Mint an NFT
+      {
```

Comment:
Q: Are the extra curly braces around each group of operations needed?

Reply:
They help to keep the code structure consistent for every transaction by creating separate closures. That allows the redeclaration of variables with the same name, e.g. `tx`, `responseCode`, etc. without overwriting over already existing variables.